app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: [:create, :destroy]
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build(micropost_params)
```

Comment:
convention: chỗ này em bỏ được cặp ngoặc tròn nhé
```suggestion
    @micropost = current_user.microposts.build micropost_params
```

Reply:
đã xóa ạ